velox/type/Timestamp.h

```diff
@@ -160,11 +161,11 @@ struct Timestamp {
     // T - literal T
     // %T - equivalent to "%H:%M:%S" (the ISO 8601 time format)
     // so this return time in the format
-    // %Y-%m-%dT%H:%M:%S.nnnnnnnnn
+    // %Y-%m-%dT%H:%M:%S.nnn
```

Comment:
This change is no longer accurate, right?

Reply:
correct. Updated message to include both formats